certified-connectors/IronDefense/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2246 @@
+{
+  "securityDefinitions": {
+    "basic": {
+      "type": "basic"
+    }
+  },
+  "security": [
+    {
+      "basic": [],
+      "jwt": []
+    }
+  ],
+  "swagger": "2.0",
+  "info": {
+    "title": "IronNet IronAPI",
+    "description": "IronNet’s IronAPI allows clients to interface with the IronDefense backend through a curated set of endpoints. This provides customers with the capability to triage alerts outside of IronVue.",
+    "version": "3.4.9",
```

Comment:
@marcus-defazio Since your connector is new and never certified, It can be `"1.0.0"`.

Reply:
changed